examples/tv-app/linux/AppImpl.cpp

```diff
@@ -168,10 +168,11 @@ CHIP_ERROR ContentAppFactoryImpl::LookupCatalogVendorApp(uint16_t vendorId, uint
     return CHIP_NO_ERROR;
 }
 
-CHIP_ERROR ContentAppFactoryImpl::ConvertToPlatformCatalogVendorApp(CatalogVendorApp sourceApp, CatalogVendorApp * destinationApp)
+CHIP_ERROR ContentAppFactoryImpl::ConvertToPlatformCatalogVendorApp(const CatalogVendorApp * sourceApp,
```

Comment:
Just to check, why not:
```suggestion
CHIP_ERROR ContentAppFactoryImpl::ConvertToPlatformCatalogVendorApp(const CatalogVendorApp & sourceApp,
```
?

Reply:
good feedback, updated to follow this convention